pkgs/applications/misc/please-cli/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, curl
+, fetchFromGitHub
+, jq
+, makeBinaryWrapper
+, please-cli
+, stdenv
+, testers
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  name = "please";
+
+  pname = "please-cli";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "TNG";
+    repo = finalAttrs.pname;
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-rJIR4eMhXL4K9iO7JxnkgWNsICV3hPQb0aobWNuHAG0=";
+  };
+
+  nativeBuildInputs = [ makeBinaryWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp $src/please.sh $out/bin/
+    wrapProgram $out/bin/please.sh \
+      --prefix PATH : ${lib.makeBinPath [ curl jq ]}
+    ln -s $out/bin/please.sh $out/bin/please
+    runHook postInstall
+  '';
+
+  passthru.tests = testers.testVersion {
+    package = please-cli;
+    command = "please --version";
+    version = "v${finalAttrs.version}";
```

Comment:
You can set `meta.mainProgram = "please";` and these should no longer be necessary.

Reply:
As `please --version` prints out the version with prefix `v` it seems to be necessary. At least that is how I read the last example in https://ryantm.github.io/nixpkgs/builders/testers/#tester-testVersion

Reply:
At least `command` can be dropped?